tests/CMakeLists.txt

```diff
@@ -791,10 +791,29 @@ foreach(prec IN LISTS TEST_PRECS)
                      --gtest_output=xml:gauge_force_test_${prec}.xml)
   endif()
 
+  if(QUDA_DIRAC_STAGGERED)
+    add_test(NAME unitarize_link_${prec}
+             COMMAND ${QUDA_CTEST_LAUNCH} $<TARGET_FILE:unitarize_link_test> ${MPIEXEC_POSTFLAGS}
+                     --dim 2 4 6 8 --prec ${prec}
+                     --gtest_output=xml:unitarize_link_test_${prec}.xml)
+  endif()
+
+  if(QUDA_FORCE_HISQ)
+    add_test(NAME hisq_paths_force_${prec}
+             COMMAND ${QUDA_CTEST_LAUNCH} $<TARGET_FILE:hisq_paths_force_test> ${MPIEXEC_POSTFLAGS}
```

Comment:
Have these been run to verify they work? iirc HISQ requires all dimensions to be >= 6, but maybe the force in isolation doesn't require that.

Reply:
It's only the improved staggered dslash that has this constraint.  The force terms don't have this limitation (since they use the extended gauge exchange for comms).

Reply:
So yes, these have been verified to work.